Mathlib/CategoryTheory/Products/Bifunctor.lean

```diff
@@ -46,4 +45,58 @@ theorem diagonal' (F : C × D ⥤ E) (X X' : C) (f : X ⟶ X') (Y Y' : D) (g : Y
     F.map (f ×ₘ 𝟙 Y) ≫ F.map (𝟙 X' ×ₘ g) = F.map (f ×ₘ g) := by
   rw [← Functor.map_comp, prod_comp, Category.id_comp, Category.comp_id]
 
+/-- Opposite of a bifunctor -/
+@[simps!]
+def biop (F : C × D ⥤ E) : Cᵒᵖ × Dᵒᵖ ⥤ Eᵒᵖ :=
+  (prodOpEquiv C).inverse ⋙ F.op
+
+/-- Flipping of arguments of a bifunctor -/
+@[simps!]
+def flip (F : C × D ⥤ E) : D × C ⥤ E :=
+  Prod.swap D C ⋙ F
+
 end CategoryTheory.Bifunctor
+
```

Comment:
Since the code below does not mention products anymore, I am not sure this file is a good home for it. I think the two-variable stuff can belong in `Mathlib.Functor.Category`, which already has some code dealing with two-variable functors.